driver/kubernetes/kubernetes.go

```diff
@@ -363,15 +370,37 @@ func (k *Driver) deleteJob(name string) error {
 }
 
 func generateNameTemplate(op *driver.Operation) string {
-	return fmt.Sprintf("%s-%s-", op.Installation, op.Action)
+	name := fmt.Sprintf("%s-%s", op.Action, op.Installation)
+	if len(name) > 50 {
+		name = name[0:39]
```

Comment:
Why does the code check for length greater than 50, but not truncate to the first 50 bytes?

Reply:
Good catch, missed this during a refactor.